homeassistant/components/switcher_kis/strings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Switcher",
```

Comment:
No need to specify title if it's the same as the integration name in the manifest